deploy/templates/notificationtemplates/appstudio/userprovisioned/notification.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>
+    Note: This is an automated email message. Please do not reply to this email. For any assistance, contact us at sscs@redhat.com.
+  </title>
+  <style>
+    a:hover {
+      text-decoration: underline !important;
+    }
+    p {
+      text-align: left;
+      margin: 30px 0;
+    }
+  </style>
+</head>
+
+<body
+        style="
+       padding: 10px;
+       padding: 0;
+       background-color: #f9f9f9;
+       font-family: 'Open Sans', sans-serif;
+       font-size: 15px;
+       font-weight: lighter;
+       line-height: 1.2;"
+>
+<div
+        style="
+       min-height: 300px;
+       max-width: 750px;
+       margin: 0 auto;
+       padding: 20px;
+       border: 1px solid #d7d7d7;
+       border-radius: 4px;
+       background-color: #fff;
+       box-shadow: 0 2px 4px #d7d7d7;"
+>
+
+  <p>
+    Dear xyz,
+  </p>
+
+  <p>
+    Welcome to Red Hat CI/CD! Red Hat CI/CD saves you weeks of time by automating the onboarding process for containerized apps with CI/CD. It also offers granular customization for pipelines, once they are running. And it provides unparalleled security for your software supply chain.
+  </p>
+
+  <p>
+    We have enabled your access, so to get started, you can now visit https://console.dev.redhat.com/hac/stonesoup and select + Create an application.
```

Comment:
We will have to parameterize the console URL. Different environments (stage, prod, etc) will have different console URLs. Let's skip the URL for now.
For now we can use something like:
```suggestion
    We have enabled your access, so to get started, you can now visit Red Hat CI/CD console and select + Create an application.
```

Reply:
done in [10a70c1](https://github.com/codeready-toolchain/host-operator/pull/754/commits/10a70c1831306e3c08c37a3fa27740c5a8d66d0b)